airflow/providers/fab/provider.yaml

```diff
@@ -32,6 +32,7 @@ source-date-epoch: 1723970140
 
 # note that those versions are maintained by release manager - do not update them manually
 versions:
+  - 1.4.0
```

Comment:
Any reason for this update?

Reply:
Yes, see this [comment](https://github.com/apache/airflow/pull/42042#issuecomment-2333313742) but I am actually not sure we need it (see my [comment why](https://github.com/apache/airflow/pull/42042#issuecomment-2334336251)). This is why I put the PR in draft, I dont want any accidental merge before sorting this out :)

Reply:
I removed it